README.md

```diff
@@ -204,16 +203,24 @@ Infer a divergence tree from the alignment.
 Augur's tree subcommand is a lightweight wrapper around existing tree builders, providing some standardization of the input alignment and output across tools.
 We use IQ-TREE by default, but other options include FastTree and RAxML.
 
+> Note: All tree builders used by Augur are maximum-likelihood (ML) tools, enabling the "real-time" part of Nextstrain’s mission at the expense of the posterior and more sophisticated models available through Bayesian methods.
+> The ML approach enables rapid prototyping to identify genomes to include in a more complex, longer-running Bayesian analysis.
+
 ``` bash
 augur tree \
   --alignment results/aligned.fasta \
   --output results/tree_raw.nwk
 ```
 
-We can view this tree and its metadata in [auspice.us](https://auspice.us/) or FigTree.
+We can view the divergence tree by loading `tree_raw.nwk` in [auspice.us](https://auspice.us/).
 
-> Note: All tree builders used by Augur are maximum-likelihood (ML) tools, enabling the "real-time" part of Nextstrain’s mission at the expense of the posterior and more sophisticated models available through Bayesian methods.
-> The ML approach enables rapid prototyping to identify genomes to include in a more complex, longer-running Bayesian analysis.
+You should see a view like this:
+
+![tree_raw.nwk in auspice.us](images/tree_raw.jpeg)
+
+Let's familiarize with the Auspice interface. The main view is an interactive phylogenetic tree. To the left are controls for the main view. There is not much going on because Newick files lack additional data that powers most of the Auspice interface. We will add those in the next steps.
```

Comment:
```suggestion
Let's familiarize ourselves with the Auspice interface. The main view is an interactive phylogenetic tree. To the left are controls for the main view. There is not much going on with this particular visualization, because Newick files lack the additional data that powers most of the Auspice interface. We will add that in the next steps.
```

Reply:
Added in d187ba0da5246af45e8e9cce1f2b2dad38259366